build-release.ini

```diff
@@ -0,0 +1,17 @@
+; SPDX-FileCopyrightText: 2024 Emil Velikov <emil.l.velikov@gmail.com>
+; SPDX-FileCopyrightText: 2024 Lucas De Marchi <lucas.de.marchi@gmail.com>
+;
+; SPDX-License-Identifier: LGPL-2.1-or-later
+
+[project options]
+build-tests = true
+debug-messages = false
+docs = true
```

Comment:
As mentioned elsewhere the gtk-doc/python regex is noisy, so let's not scare/scare the normal folks and switch this off as well?